torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -228,17 +272,49 @@ def _wrap_fx_args_as_onnxscript_args(
         if isinstance(arg, torch.fx.Node):
             # Create a concreate test tensor based on the fake tensor
             with torch.utils._mode_utils.no_dispatch():
-                # TODO(titaiwang): The assumption of torch.float might not be true, eg: aten_where needs BOOL in input_args
+                # TODO(titaiwang): The assumption of torch.float might not be true,
+                # https://github.com/microsoft/onnx-script/issues/393
+                # eg: aten_where needs BOOL in input_args
                 # fx_name_to_onnxscipt_value could help?
-                if isinstance(arg.meta["val"], list):
-                    for meta_value in arg.meta["val"]:
-                        torch_args.append(
-                            torch.randn_like(meta_value, dtype=torch.float)
+                fake_tensor = arg.meta["val"]
+                if isinstance(fake_tensor, list):
```

Comment:
Too hacky...and it will go wrong when there are shape-realted ops. Let's not to verify when dynamic axis is on.

Reply:
This is actually for op level debugging, I will split this part out.